code/modules/mob/living/carbon/xenomorph/attack_alien.dm

```diff
@@ -956,6 +956,8 @@
 		SPAN_DANGER("We slice [src] apart!"), null, 5, CHAT_TYPE_XENO_COMBAT)
 		playsound(loc, 'sound/effects/metalhit.ogg', 25, TRUE)
 		dismantle()
+	if(state == STATE_DESTROYED)
+		qdel(src)
```

Comment:
Girders are meant to be repaired after slashed . Not permanently destroyed

Reply:
Balance tag

Reply:
they are still repairable, as they only appear after you break a wall. This doesnt really change anything